talk/owt/sdk/base/customizedoutputaudiodevicemodule.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) <2023> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OWT_BASE_CUSTOMIZEDOUTPUTAUDIODEVICEMODULE_H_
+#define OWT_BASE_CUSTOMIZEDOUTPUTAUDIODEVICEMODULE_H_
+
+#include <mutex>
+#include "third_party/webrtc/modules/audio_device/audio_device_buffer.h"
+#include "third_party/webrtc/modules/audio_device/include/fake_audio_device.h"
+#include "third_party/webrtc/rtc_base/platform_thread.h"
+
+namespace owt {
+namespace base {
+class CustomizedOutputAudioDeviceModule : public webrtc::FakeAudioDeviceModule {
+ public:
+  CustomizedOutputAudioDeviceModule();
+  int32_t RegisterAudioCallback(webrtc::AudioTransport* audioCallback) override;
+  int32_t StopPlayout() override;
+  int32_t PlayoutIsAvailable(bool* available) override;
+  int32_t InitPlayout() override;
+  int32_t StartPlayout() override;
+  bool Playing() const override;
+  bool Recording() const override;
+  int32_t StereoPlayoutIsAvailable(bool* available) const override;
+  int32_t StereoRecordingIsAvailable(bool* available) const override;
+
+ private:
+  bool PlayThreadProcess();
+
+  std::unique_ptr<webrtc::TaskQueueFactory> task_queue_factory_;
+  std::unique_ptr<webrtc::AudioDeviceBuffer> audio_device_buffer_;
+  std::unique_ptr<rtc::PlatformThread> play_thread_;
+  size_t playout_frames_in_10ms_;
+  bool playing_;
```

Comment:
initialize in header instead of using initialization list for this.

Reply:
Because `task_queue_factory_`, `audio_device_buffer_`, `play_thread_` are using initialization list, I'm wondering is there any specific reason to initialize these two in header?

Reply:
Done. As playing_ and last_call_millis_ are simple members, we agree to initialize them in header.